headers/Instruction.hpp

```diff
@@ -15,11 +15,19 @@
 
 #include "headers/UID.hpp"
 #include "headers/Enums.hpp"
+
+#ifdef _WIN32
+#define DEBUG_BREAK __debugbreak()
+#else
+#include <signal.h>
+#define DEBUG_BREAK raise(SIGTRAP);
+#endif
+
 namespace PLH {
 class Instruction {
 public:
 	union Displacement {
-		int64_t  Relative;
+		int64_t  Relative = 0;
```

Comment:
why

Reply:
just to make sure it's always initialised